app/javascript/store/modules/froggo_team/actions.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+import {
+    CREATE_NEW_FROGGO_TEAM,
+} from '../../action-types';
+
+export default {
+  [CREATE_NEW_FROGGO_TEAM](_,{ name, organization_id, users }) {
+    axios.post(
+      `/api/organizations/${organization_id}/froggo_teams`,
+      { name },
+      {headers: {"Content-Type": "application/json"} },
```

Comment:
A space is required after '{'                                                                                  object-curly-spacing<br>Strings must use singlequote                                                                                   quotes<br>A space is required before '}'                                                                                 object-curly-spacing